### DISCUSSION TITLE
import 'vue-currency-input' has a error in compiling

### DISCUSSION THREAD

**Discussion by KKConney** [dm4t2/vue-currency-input#282] [Q&A]
Hi!  I import this component(V2) to use in my project(vue2.0, no vue3),  but "Failed to compile",  See following chart：
<img width="1175" alt="WX20210928-161836@2x" src="https://user-images.githubusercontent.com/8678483/135050276-0e9fb788-a185-4dfe-9d16-07bcaa57391c.png">

**Comment by dm4t2** [comment]
Hi, can you provide a minimal reproduction repo? Thanks!
